client/src/components/protected/styleMatch.js

```diff
@@ -1,89 +1,80 @@
+import { emphasize } from "@material-ui/core/styles";
+
 export default (theme) => ({
   centerText: {
-    textAlign: 'center',
-    marginBottom: "0.5em"
+    textAlign: "center",
   },
   leftText: {
-    textAlign: 'left'
+    textAlign: "left",
   },
   gridContainer: {
     flexWrap: "wrap",
     justifyContent: "space-around",
-    margin: "10px auto"
-  },
-  standardFlex: {
-    display: 'flex',
-    flexWrap: 'wrap',
-    // height: '80%'
+    margin: "10px auto",
   },
   matchStyle: {
     display: "flex",
     flexDirection: "row",
-    minHeight: "100vh",
-    maxHeight: "100vh",
-    justifyContent: 'left',
+    maxHeight: "90vh",
+    justifyContent: "left",
   },
-  flexRow: {
-    margin: "10px",
-    justifyContent: 'space-evenly',
-    '&>.MuiGrid-item': {
-      '&>.button': {
-        color: "#000000",
-        backgroundColor: '#ffffff',
-        width: '100%',
-        height: '100%',
-      },
-      '&>.button:hover': {
-        transform: "scale(1.3)"
-      },
-      "&>.chosenB": {
-        backgroundColor: '#00008b',
-        color: '#ffffff',
-        width: '100%',
-        height: '100%',
-      },
-      "&>.chosenR": {
-        backgroundColor: '#8b0000',
-        color: '#ffffff',
-        width: '100%',
-        height: '100%',
-      },
-      "&>.chosenA": {
-        backgroundColor: '#000000',
-        color: '#ffffff',
-        width: '100%',
-        height: '100%',
-      },
+  grid: {
+    display: "grid",
+    gridTemplate: "repeat(5, 1fr) / repeat(5, 1fr)",
 
-      "&>.chosenC": {
-        backgroundColor: '#ffcc99',
-        color: '#000000',
-        width: '100%',
-        height: '100%',
-      },
+    width: "75%",
+    height: "75%",
+  },
+  gridElement: {
+    display: "flex",
+    justifyContent: "center",
+    // width: "100%",
+    alignItems: "center",
+    margin: "5px",
+    "&:hover": {
+      transform: "scale(1.3)",
     },
-
-    '&>.MuiTypography-h5': {
-      display: "flex",
-      width: "50%",
-      height: "200px",
-      margin: "auto",
-      borderRadius: "10px",
-      border: "3px dashed #1c87c9",
+    borderRadius: "5%",
+    cursor: "pointer",
+    padding: "5%",
+    justifyContent: "space-evenly",
+    "&>.button": {
+      color: "#000000",
+      backgroundColor: "#ffffff",
+      width: "100%",
+      height: "100%",
     },
```

Comment:
This class is applied to each tile in the grid. The padding is flexible, but the margin is fixed (you don't want it to be to small)